server/src/main/resources/schema/wildlfy-product-info_1_0.xsd

```diff
@@ -0,0 +1,193 @@
+<?xml version="1.0"?>
+<!--
+Copyright (C) 2015 Red Hat, inc., and individual contributors
+as indicated by the @author tags. See the copyright.txt file in the
+distribution for a full listing of individual contributors.
+
+This library is free software; you can redistribute it and/or
+modify it under the terms of the GNU Lesser General Public
+License as published by the Free Software Foundation; either
+version 2.1 of the License, or (at your option) any later version.
+
+This library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public
+License along with this library; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+MA 02110-1301  USA
+-->
+
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
```

Comment:
I haven't checked, but this can't get into any build. Better if it gets deleted in the 2nd commit you have.